### PR TITLE
add pip package environs in 'rosdep/python.yaml'

### PR DESCRIPTION
**Pull request by yanrayw**
Please add the following dependency to the rosdep database.

## Package name:

pip packages: `environs`

## Package Upstream Source:

`environs`: https://pypi.org/project/environs/

## Purpose of using this:

For some [autoware.auto](https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto) users, if they'd like to use [lgsvl-pythonAPI](https://github.com/lgsvl/PythonAPI) to control vehicle. There are some required packages to be installed first. Pip package `environs` is one of them. To simplify dependency installation process, adding them in rosdep would help a lot. Typically, only Ubuntu and debian system are needed. So I only add them in this pull request.

This dependency is being used for this reason. This is why I think it's valuable to be added to the rosdep database. If there are any questions, please let me know. Thanks a lot!

# Please Add This Package to be indexed in the rosdistro.

[rosdep/python.yaml](https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml)

# Checks
 - [ ] All packages have a declared license in the package.xml
 - [ ] This repository has a LICENSE file
 - [ ] This package is expected to build on the submitted rosdistro

### CODE REVIEW

**Comment by YanrayWang** [comment]
Hi, @clalancette , I created a new pull request to describe in more detail of my pull request. But it looks like it requires an approval from maintainer to run two checks. Please have a look. Thanks a lot!

File: rosdep/python.yaml
Comment (clalancette) [inline rosdep/python.yaml:5334]: So we already have the [python3-websocket](https://github.com/ros/rosdistro/blob/67cea41a94e36ab0b43c85cddf15a12a7681bb67/rosdep/python.yaml#L7456-L7462) key.  Since we generally prefer the system packages, I think it would be much better to add the arch entry (`python-websocket-client`) there.

File: rosdep/python.yaml
Comment (clalancette) [inline rosdep/python.yaml:5953]: Please also add at least `debian` here.

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rosdep/python.yaml
Comment (YanrayWang) [reply rosdep/python.yaml:5953]: Thanks, added.

**Comment by YanrayWang** [comment]
Hi, @clalancette . I think `websocket-client` has been satisfied by `python3-websocket`. So I don't have to add them anymore. In addition, `ubuntu` is enough for me, no `arch` needed. Thus the only one needed to be installed is `envirions` of pip package. Thanks a lot!

**Review by clalancette — APPROVED** [review]

**Review by jacobperron — APPROVED** [review]

**Pull request merged**
